README.txt

```diff
@@ -12,22 +12,30 @@ and then use conda to install the following packages
 *skimage
 *matplotlib
 
+Input must be a binary array with z in it's first dimension
```

Comment:
it's -> its

(a very strange part of English grammar that a lot of native speakers get wrong)